### ISSUE TITLE
PolyInterpMixin<time>::fas() function question

### ISSUE THREAD

**Issue by MyronZhen** [Parallel-in-Time/PFASST#249]
The function PolyInterpMixin<time>::fas(...) in file poly_interp_impl.hpp is the function to compute FAS correction 'tau' iterm. In this function, the matrix fmat seems to have rows like [ -1 -1 -1 1 | 1 1 1 -1], which is confusing. I thought this should be like [0 0 -1 1 | 0 0 1 -1] to compute an interval quadrature. There should be something I misunderstood, so please give me some clue. Looking forward to any reply.

**Comment by pancetta** [comment]
H! Thanks for reaching out. This project is somewhat outdated, I have to admit. Did you verify that `fmat` has these rows? The loop looks confusing and redundant in parts, so I'm not sure this is what you get. Did it work (better) with your `fmat`?

**Comment by MyronZhen** [comment]
Thank you for your reply. I have tried with my `fmat` and obtained similar results as the original `fmat`, so the influence is essential but not huge. Currently I am using another code to perform PFASST, and I found I should follow the updated PFASST method rather than the original paper. Therefore, I could misunderstood your code and I am still studing the new code and figuring out the algorithm. Again, thanks for you reply!
